Snowflake.Data.Tests/IntegrationTests/MaxLobSizeIT.cs

```diff
@@ -0,0 +1,423 @@
+                Assert.AreEqual(t_colData[2], reader.GetString(2));
+                CheckColumnMetadata(reader);
+            }
+        }
+
+        private void GetFile(DbConnection conn)
+        {
+            using (var command = conn.CreateCommand())
+            {
+                // arrange
+                command.CommandText = $"GET @%{t_tableName}/{t_fileName} file://{s_outputDirectory}";
+
+                // act
+                var reader = command.ExecuteReader();
+
+                // assert
+                Assert.IsTrue(reader.Read());
+                Assert.AreEqual(ResultStatus.DOWNLOADED.ToString(),
+                    reader.GetString((int)SFResultSet.PutGetResponseRowTypeInfo.ResultStatus));
+
+                // arrange
+                var content = File.ReadAllText(t_outputFilePath).Split(',');
+
+                // assert
+                Assert.AreEqual(t_colData[0], content[0]);
+                Assert.AreEqual(t_colData[1], content[1]);
+                Assert.AreEqual(t_colData[2], content[2]);
+            }
+        }
+
+        private void AlterSessionSettings(SnowflakeDbConnection conn)
+        {
+            using (var command = conn.CreateCommand())
+            {
+                // Alter session result format
+                command.CommandText = $"ALTER SESSION SET DOTNET_QUERY_RESULT_FORMAT = {_resultFormat}";
+                command.ExecuteNonQuery();
+
+                //// Alter session max lob
+                //command.CommandText = "ALTER SESSION SET FEATURE_INCREASED_MAX_LOB_SIZE_IN_MEMORY = 'ENABLED'";
+                //command.ExecuteNonQuery();
+            }
+        }
+
+        private void CheckColumnMetadata(DbDataReader reader)
+        {
+            var dataTable = reader.GetSchemaTable();
+
+            DataRow row = dataTable.Rows[0];
+            Assert.AreEqual(s_colName[0], row[SchemaTableColumn.ColumnName]);
+            Assert.AreEqual(0, row[SchemaTableColumn.ColumnOrdinal]);
+            Assert.AreEqual(MaxLobSize, row[SchemaTableColumn.ColumnSize]);
+            Assert.AreEqual(SFDataType.TEXT, (SFDataType)row[SchemaTableColumn.ProviderType]);
+
+            row = dataTable.Rows[1];
+            Assert.AreEqual(s_colName[1], row[SchemaTableColumn.ColumnName]);
+            Assert.AreEqual(1, row[SchemaTableColumn.ColumnOrdinal]);
+            Assert.AreEqual(MaxLobSize, row[SchemaTableColumn.ColumnSize]);
+            Assert.AreEqual(SFDataType.TEXT, (SFDataType)row[SchemaTableColumn.ProviderType]);
+
+            row = dataTable.Rows[2];
+            Assert.AreEqual(s_colName[2], row[SchemaTableColumn.ColumnName]);
+            Assert.AreEqual(2, row[SchemaTableColumn.ColumnOrdinal]);
+            Assert.AreEqual(0, row[SchemaTableColumn.ColumnSize]);
+            Assert.AreEqual(SFDataType.FIXED, (SFDataType)row[SchemaTableColumn.ProviderType]);
+        }
+        private static string GenerateRandomString(int sizeInBytes)
```

Comment:
let's add a new line between the methods

Reply:
Ok